packages/hooks/src/__tests__/useCheck10kLimit.spec.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import type { TStores } from '@deriv/stores';
+import { renderHook } from '@testing-library/react-hooks';
+import useCheck10kLimit from '../useCheck10kLimit';
+import useMaxWithdrawAmount from '../useMaxWithdrawAmount';
+
+jest.mock('../useMaxWithdrawAmount');
+
+describe('useCheck10kLimit', () => {
+    test('should be false if useMaxWithdrawAmount is undefined', async () => {
+        (useMaxWithdrawAmount as jest.Mock).mockReturnValue('undefined');
+        const wrapper = ({ children }: { children: JSX.Element }) => (
+            <StoreProvider store={mockStore({}) as TStores}>{children}</StoreProvider>
+        );
+        const { result } = renderHook(() => useCheck10kLimit(), { wrapper });
+
+        expect(result.current).toBe(false);
+    });
+    test('should be false if useMaxWithdrawAmount is bigger than min_withdrawal', async () => {
```

Comment:
```suggestion
    });
    
    test('should be false if useMaxWithdrawAmount is bigger than min_withdrawal', async () => {
```
@hamid-deriv Just some spacing for better readability, Would you please add it for other places as well 🙏🏻

Reply:
Sorted.